### PR TITLE
[EasyDoctrine] Move TimestampableImmutableTrait from AVC

### PR DESCRIPTION
**Pull request by skrut**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

### CODE REVIEW

**Review by akl-95 — CHANGES_REQUESTED** [review]
Please move also interface and subscriber.

**Review by akl-95 — CHANGES_REQUESTED** [review]
I think we have to change namespaces. Timestampable-related classes should be either in one `Timestampable` directory (like in AVC) or we have to split all classes by type (Trait, Interface, Subscriper).
What do you think guys?

**Comment by roman-eonx** [comment]
> I think we have to change namespaces. Timestampable-related classes should be either in one `Timestampable` directory (like in AVC) or we have to split all classes by type (Trait, Interface, Subscriper). What do you think guys?

Let's keep the same structure as in other Easy-packages and introduce a specific namespace for traits and interfaces.

Please also move `EntityEventSubscriberInterface` to the new namespace.

File: packages/EasyDoctrine/src/Timestampable/TimestampableImmutableTrait.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyDoctrine\Timestampable;
+
+use Carbon\CarbonImmutable;
```
Comment (roman-eonx) [inline packages/EasyDoctrine/src/Timestampable/TimestampableImmutableTrait.php:7]: You have to add Carbon to `composer.json` this way:
```
"nesbot/carbon": "^1.39 || ^2.22"
```

**Review by roman-eonx — CHANGES_REQUESTED** [review]

**Review by akl-95 — DISMISSED** [review]

File: packages/EasyDoctrine/src/Timestampable/TimestampableImmutableTrait.php (hunk 1 above)
Comment (skrut) [reply packages/EasyDoctrine/src/Timestampable/TimestampableImmutableTrait.php]: done

**Comment by skrut** [comment]
> 

Done, but for now EonX\\EasyDoctrine\\Subscribers\\EntityEventSubscriberInterface not found#' in `EasyActivity`

**Comment by akl-95** [comment]
> Done, but for now EonX\EasyDoctrine\Subscribers\EntityEventSubscriberInterface not found#' in `EasyActivity`
Yes, but we can change the namespace in `EasyActivity` too.

**Comment by skrut** [comment]
> > Done, but for now EonX\EasyDoctrine\Subscribers\EntityEventSubscriberInterface not found#' in `EasyActivity`
> 
> Yes, but we can change the namespace in `EasyActivity` too.

indeed, done

**Review by roman-eonx — APPROVED** [review]

**Review by akl-95 — APPROVED** [review]

**Review by mvlad — APPROVED** [review]

**Review by natepage — APPROVED** [review]

**Pull request merged**